client/state/login/selectors.js

```diff
@@ -164,7 +158,7 @@ export const getRequestNotice = ( state ) => {
  * @see getRedirectToSanitized for the sanitized version
  */
 export const getRedirectToOriginal = ( state ) => {
-	return get( state, 'login.redirectTo.original', null );
+	return state.login.redirectTo?.original ?? null;
```

Comment:
The `redirectTo` reducer is a result of `combineReducers`, is always a valid object, and always has the `.original` and `.sanitize` properties. No need to optionally chain `redirectTo?.original`.

Reply:
Good point, `combineReducers` does mean it'll always be available in this case. Done.